### DISCUSSION TITLE
terminate called after throwing an instance of 'nlohmann::detail::type_error'   what():  [json.exception.type_error.302] type must be number, but is string Milvus server is going to shutdown ...

### DISCUSSION THREAD

**Discussion by Nebuverse** [milvus-io/milvus#4297] [Q&A and General discussion] [answered]
our index type is FLAT, our data size is 100w, when we search the a picture(pic search system), we We ran into a problem. it is followed.
![image](https://user-images.githubusercontent.com/45379784/100328332-5c842300-3007-11eb-9445-e294eb62911b.png)

The environment we use is：
server_version:  0.10.3
client_version:  0.2.14

server_config follow

version: 0.5

cluster:
  enable: false
  role: rw

general:
  timezone: UTC+8
  meta_uri: mysql://root:123456@5.5.3.223:3306/milvus

network: 
  bind.address: 0.0.0.0
  bind.port: 19530
  http.enable: true
  http.port: 19121

storage:
  path: /var/lib/milvus
  auto_flush_interval: 1

wal:
  enable: true
  recovery_error_ignore: false
  buffer_size: 1024MB
  path: /var/lib/milvus/wal

cache:
  cache_size: 30GB
  insert_buffer_size: 20GB
  preload_collection:

gpu:
  enable: true
  cache_size: 14GB
  gpu_search_threshold: 2000
  search_devices:
    - gpu0
  build_index_devices:
    - gpu0

logs:
  level: debug
  trace.enable: true
  path: /var/lib/milvus/logs
  max_log_file_size: 1024MB
  log_rotate_num: 0

metric:
  enable: false
  address: 127.0.0.1
  port: 9091

**Comment by shiyu22** [comment, marked as answer]
Hi,
We have responded to you in [issue4298](https://github.com/milvus-io/milvus/issues/4298).
Sincerely.
